codecarbon/output.py

```diff
@@ -75,18 +75,16 @@ class HTTPOutput(BaseOutput):
     Send emissions data to HTTP endpoint
     """
 
-    def __init__(self, endpoint_url: str):
+    def __init__(self, endpoint_url: str, experiment_id: str, api_key: str):
         self.endpoint_url: str = endpoint_url
+        self.api = ApiClient(
+            experiment_id=experiment_id,
+            endpoint_url=endpoint_url,
+            api_key=api_key,
+        )
 
     def out(self, data: EmissionsData):
         try:
-            payload = dataclasses.asdict(data)
```

Comment:
I think we should keep this for people wanting to push to another system, which would mean to have another class `APIOutput` or something. Thoughts?

Reply:
OK, I've created a CodeCarbonAPIOutput class and leave a model for HTTPOutput.